src/hazelcore/Outcome.re

```diff
@@ -0,0 +1,9 @@
+type t('success) =
```

Comment:
Let's call this module ActionOutcome, rather than just Outcome, to make it more clear that it is related to actions